common/lib/xmodule/xmodule/tests/test_tabs.py

```diff
@@ -735,6 +744,7 @@ def test_tabs_with_discussion(self):
             tab_list=self.tabs_with_discussion,
             expected_discussion_link="default_discussion_link",
             expected_can_display_value=True,
+            is_hideable=True,
```

Comment:
@smarnach nit: I think I woud prefer if the ability to hide the tab was tested in a separate test case, similar to https://github.com/open-craft/edx-platform/blob/82d2c8a0e38812f41b9b7c3ffd0c21b4734d332d/common/lib/xmodule/xmodule/tests/test_tabs.py#L221-L231 rather than testing it inside `check_discussion`.